src/content/pages/privacy.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Privacy policy
+metaDescription: We care about your privacy! Find out more about how your data is used to make Turnip better.
+---
+
+import { Link, Paragraph } from '@octopusthink/nautilus'
+
+import Panel from 'components/Panel'
+
+<Panel>
+
+# Privacy policy
+
+Turnip is a simple timer app made by Sarah &amp; Matt at [Octopus Think](https://octopusthink.com/about).
+
+By default, Turnip collects **absolutely no data** about you, your computer, or your usage of the app.
+
+## Beta programme
+
+If you sign up for the Turnip beta, crash reporting and analytics are mandatory. This is to ensure that we know if anything is going wrong. Your data is anonymised and isn’t tied to any personal identifier in any way. If that makes you uncomfortable, we totally understand, but we think this is the best way to ensure that we can improve the app for our beta users as quickly and efficiently as possible.
+
+## Integrations
+
+Turnip connects to your third-party services using secure authentication tokens provided by the third-party service. We don't store, collect, or even see your password or login credentials for any third-party service. 
```

Comment:
This phrasing is _technically_ true, but it sort of makes it sound like we don't store those obtained authentication tokens. It's also quite technical and not really about privacy, more about security.

For what it's worth, I suspect we _will_ store OAuth tokens for things like FreeAgent in the future so we can sync them for users and save them the hassle of having to sign in again on a new Mac, etc.

```suggestion
Turnip connects to your third-party services using secure authentication tokens provided by the third-party service. We don't store, collect, or even see your password or login credentials for any third-party service. 
```

Reply:
I think I'm mostly trying to say "we don't save your password" here, and clarify that we don't collect data associated with their <whatever> account. 

Is this the suggested change you meant to make? I've compared them like six times and I really can't find a difference! 😅